src/components/limited-restoration-panel/LimitedRestorationPanel.vue

```diff
@@ -112,4 +123,8 @@ export default class LimitedRestorationPanel extends Mixins(DateMixin) {
 .month-text {
   color: $gray7;
 }
+
+.radio-button {
+    padding-top: 15px;
+}
```

Comment:
If you inspect the rendered HTML for the radio buttons, you will find that Vuetify already adds classnames that you can select on, so you don't have to add your own generic classname,

You may need to use `:deep(your_selector_here)` to target the rendered HTML.

Also, I know the UX team is asking for whitespace of 15 px, but since we're using Vuetify, everything is actually in quarter-rems (eg. pt-1 is "padding top 4px"). Can you push back a bit so we can have a padding value that is a quarter-rem?

Reply:
Thank you so much for this. I searched everywhere for how to change the label color for a v-radio and I saw /deep/ but it was in CSS. I couldn't figure it out, so I created all these slots. This makes my life easier now thanks!

Noted. I will be working on that.